modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney_controller.rb

```diff
@@ -12,7 +12,7 @@ class PowerOfAttorneyController < ClaimsApi::V2::ApplicationController
 
         def show
           poa_code = BGS::PowerOfAttorneyVerifier.new(target_veteran).current_poa_code
-          head(:no_content) && return if poa_code.blank?
+          raise ::Common::Exceptions::ResourceNotFound.new(detail: 'POA not found') if poa_code.blank?
 
```

Comment:
I think we probably want to use our new Lighthouse error handler here for the resource not found error.  That is a somewhat new thing we developed for 526 so that our error returns where in line with Lighthouse Standards.  In order to use that here you would want something along the lines of the attached image (require, include and then using it for the error).  It looks like some updates from master are not in your branch though, one I specifically noticed was in the `lighthouse_error_handler.rb` there was an update for the pointer value to shorten it and your current branch does not have that, not sure why but you would want that in there before compiling the docs.  

![Screenshot 2024-01-04 at 10 08 43 AM](https://github.com/department-of-veterans-affairs/vets-api/assets/129893414/f8e4f288-d615-4b63-ba91-b9d0f59b579f)

Reply:
You would then get an error like this returned (and in the docs), which is inline with lighthouse standards
![Screenshot 2024-01-04 at 10 21 51 AM](https://github.com/department-of-veterans-affairs/vets-api/assets/129893414/3b41b503-727e-417f-9b7d-06c0c52993da)